### PR TITLE
OS 182 cassandra support changes

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

File: java/registry/src/main/resources/application.yml.sample [hunk 1]
```diff
@@ -53,7 +53,7 @@ database:
 
   # Providers available are NEO4J, SQLG, CASSANDRA, ORIENTDB, TINKERGRAPH (in-memory).
   # NOTE: Values given against 'shardId' must be unique
-  provider: ${database_provider:NEO4J}
+  provider: ${database_provider:CASSANDRA}
```
Comment (indrajra) [inline java/registry/src/main/resources/application.yml.sample]: Please do not change the default to cassandra. Let the example connection_uri be there and lets check in.

File: java/registry/src/main/java/io/opensaber/registry/service/impl/RegistryServiceImpl.java [hunk 2]
```diff
@@ -260,15 +260,15 @@ public void updateEntity(String id, String jsonString) throws Exception {
             logger.debug("After merge the payload is " + mergedNode.toString());
 
             // Re-sign, i.e., remove and add entity signature again
-            if (signatureEnabled) {
+            /*if (signatureEnabled) {
```
Comment (indrajra) [inline java/registry/src/main/java/io/opensaber/registry/service/impl/RegistryServiceImpl.java]: Investigation required if the value of "signatureFor" causes any problem.

File: java/registry/src/main/resources/application.yml.sample (hunk 1 above)
Comment (Hari-stackroute) [reply java/registry/src/main/resources/application.yml.sample]: for the connection_uri changes are done, actually database_provider should be changed to cassandra. Is my understanding right, on your ask?

File: java/registry/src/main/java/io/opensaber/registry/service/impl/RegistryServiceImpl.java (hunk 2 above)
Comment (Hari-stackroute) [reply java/registry/src/main/java/io/opensaber/registry/service/impl/RegistryServiceImpl.java]: Faced a problem with JanusGraph api, vertex property cannot be labeled to any edge. Did modifications accordingly.

**Pull request merged**